frontend/src/pages/sign-in.jsx

```diff
@@ -46,7 +46,7 @@ function SignIn() {
     if (localStorage.getItem('userId') != null) {
       navigate('/user');
     }
-  }, []);
+  }, [navigate]);
```

Comment:
just wondering what is the reason for this? :)

Reply:
ahh cuz i noticed a warning in frontend/ when compiling which says that navigate should be added as a depedency. after the change the page still works so i just left it like that :""

Reply:
Ah okay i can do that for the other warnings too then